### DISCUSSION TITLE
Provisioning Library-Elements via Filesystem

### DISCUSSION THREAD

**Discussion by guusvw** [grafana/grafana#44388] [Dashboards, Panels & Visualizations]
Grafana allows it to [provision resources](https://grafana.com/docs/grafana/latest/administration/provisioning/) like Dashboards, Plugins, ... via the Filesystem and also allows it to let Grafana reload such resources during runtime via an HTTP endpoint.
Would it be also possible to add the capabilities for [library-elements](https://grafana.com/docs/grafana/latest/http_api/library_element/#create-library-element)?

We're making use of this mechanism already for Dashboards and it would be great to use the same pattern for Library-Elements instead of the HTTP endpoints for creating/updating/deleting the Library-Elements.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
